docs/structure-of-a-contract.rst

```diff
@@ -123,6 +123,16 @@ Structs are custom defined types that can group several variables (see
             uint vote;
         }
     }
+    
+If you want to create a new struct variable inside a function, 
```

Comment:
I think this should rather go here: https://solidity.readthedocs.io/en/v0.4.23/types.html#structs